site/content/usage/experimental/01-gitops.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "GitOps"
+weight: 10
+url: usage/experimental/gitops-flux
+---
+
+## GitOps
+
+[GitOps][gitops] is a way to do Kubernetes application delivery. It works by using Git as a single source of truth for
+Kubernetes resources. With Git at the center of your delivery pipelines, developers can make pull requests to accelerate
+and simplify application deployments and operations tasks to Kubernetes.
+
+`eksctl` provides an easy way to set up gitops in an existing cluster with the `eksctl install flux` command.
+
+[gitops]: https://www.weave.works/technologies/gitops/
+
+
+### Installing Flux
+
+> This is an experimental feature. To enable it, set the environment variable `EKSCTL_EXPERIMENTAL=true`. 
+> Experimental features are not stable and their command name and flags may change.
+
+Installing Flux on the cluster is the first step towards a gitops workflow. To install it, you need an empty repo and
+an existing cluster. Then run the following command:
+
+```
+EKSCTL_EXPERIMENTAL=true eksctl install flux --name <cluster_name> --region <region> --git-url=<git_repo> --git-email=<git_user_email>
+```
+
+Or use a config file:
+```bash
+EKSCTL_EXPERIMENTAL=true eksctl install flux -f examples/01-simple-cluster.yaml --git-url=git@github.com:weaveworks/cluster-1-gitops.git --git-email=johndoe+flux@weave.works
```

Comment:
Shouldn't we use the same template arguments here? (e.g. `<region>`).

Also, shouldn't we use the `bash` syntax-highlighting in the previous command?

Reply:
The region is specified in the config file.

I didn't ad the bash highlighting before because goland was hinting that that line is actually not bash. It turns out that it does a better job at highlighting the right thing without it in that case.